_config.yml

```diff
@@ -21,6 +21,8 @@ description:
   gr: Μαθήματα προγραμματισμού της γλώσσας Elixir
   it: Lezioni sul linguaggio di programmazione Elixir
 url: https://elixirschool.com
+insecure_url: http://elixirschool.com #Evernote gives a 500 error on the https
```

Comment:
There may be an alternative way of doing this, but each time I try to "clip" the https site, it gives an error:

![image](https://cloud.githubusercontent.com/assets/22348/14304759/b7dfa05c-fb83-11e5-9ffe-71fa936f229b.png)